src/renderer/i18n/en.js

```diff
@@ -127,6 +127,23 @@ const English = {
       onlyCustom: "Use custom layers only",
       defaultLayer: "Default layer"
     },
+    led: {
+      brightness: "Adjust LED brightness",
+      idleDisabled: "Disabled",
+      idleTimeLimit: "Idle time before LEDs turn off",
+      idle: {
+        oneMinute: "1 minute",
+        twoMinutes: "2 minutes",
+        threeMinutes: "3 minutes",
+        fourMinutes: "4 minutes",
+        fiveMinutes: "5 minutes",
+        tenMinutes: "10 minutes",
+        fifteenMinutes: "15 minutes",
+        twentyMinutes: "20 minutes",
+        thirtyMinutes: "30 minutes",
+        oneHour: "1 hour"
```

Comment:
Does our I18N library have placeholder support? So we can do "%d minutes"?

Reply:
We don't really have an i18n library. We have translated strings. :(

(And nope, it doesn't have `%d minutes`. We should probably change that in the not too distant future, and use a real i18n library.)